src/components/react-admin/MapPointInput.js

```diff
@@ -34,10 +33,21 @@ const MapPointInput = ({ classes, input }) => {
     customProps.center = input.value;
   }
 
+  React.useEffect(() => {
+    if (mapConfig.accessToken) {
+      Map = ReactMapboxGl({
+        accessToken: mapConfig.accessToken,
+      });
+      setLoaded(true);
+    }
+  }, [mapConfig.accessToken]);
+
   return (
+    (loaded && (
```

Comment:
There's a pair of extraneous parents:
You have `return (loaded && (<Map>...</Map>));` where `return loaded && (<Map>...</Map>);` would be enough.

Although, I would personaly prefer this :
```js
if (!loaded) {
  return null; // Or maybe `return <Loading />`;
}

return (
  <Map
    ...
  >
  ...
  </Map>
);
```